account/account.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package account
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/key"
+)
+
+type Account struct {
```

Comment:
I think we should explain to the user in a doc here what this abstraction mean and why we use it